app/celery/scheduled_tasks.py

```diff
@@ -207,14 +209,38 @@ def replay_created_notifications():
 
 @notify_celery.task(name='check-if-letters-still-pending-virus-check')
 def check_if_letters_still_pending_virus_check():
-    letters = dao_precompiled_letters_still_pending_virus_check()
+    letters = []
+
+    for letter in dao_precompiled_letters_still_pending_virus_check():
+        # find letter in the scan bucket
+        filename = generate_letter_pdf_filename(
+            letter.reference,
+            letter.created_at,
+            ignore_folder=True,
+            postage=letter.postage
+        )
+
+        if s3.file_exists(current_app.config['LETTERS_SCAN_BUCKET_NAME'], filename):
+            current_app.logger.warning(
+                f'Letter id {letter.id} got stuck in pending-virus-check. Sending off for scan again.'
+            )
+            notify_celery.send_task(
+                name=TaskNames.SCAN_FILE,
+                kwargs={'filename': filename},
+                queue=QueueNames.ANTIVIRUS,
+            )
+        else:
+            letters.append(letter)
 
     if len(letters) > 0:
```

Comment:
If there is a letter here, we could kick off the app.celery.letters_pdf_tasks.process_sanitised_letter, right?

Reply:
hmmm, i'm keen to think about other failure cases, but `process_sanitised_letter` expects the letter to be in the scan bucket. so that task would just fail.

I think it's fine to keep as is. The letter might have passed the virus scan and fallen over in the `sanitise_letter` task, but doing another virus scan on it won't hurt.

Reply:
oh right, it looks in the scan bucket too. Fair enough.